File: Assignment8/C/Program4.c
// PS - Write a program which acepts one number from user and return multiplication of all digits

#include<stdio.h>

int Multiplication(int iNo) {
    int iDigit = 0;
    int iMult = 1;

    if(iNo < 0) {
        iNo = -iNo;
    }

    while(iNo != 0) {
        iDigit = iNo % 10;
        iMult = iDigit * iMult;
        iNo = iNo / 10;
    }
    return iMult;
}

int main() {
    int iValue = 0;
    int iRet = 0;

    printf("Enter number \n");
    scanf("%d", &iValue);

    iRet = Multiplication(iValue);

    printf("Multiplication of all digits is %d \n", iRet);

    return 0;
}

File: Assignment35/Java/Program4.java
 // PS - Write a java program which accept number from user and return multiplication of all digits

import java.util.*;

class DigitDemo
{
    public int Multiplication(int No)
    {
        int Mult = 1;
        int iDigit = 0;

        while(No != 0)
        {
            iDigit = No % 10;
            Mult = Mult * iDigit;
            No = No / 10;
        }
        return Mult;
    }
}

class Program4
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        int No = sobj.nextInt();

        DigitDemo dobj = new DigitDemo();
        int iRet = dobj.Multiplication(No);

        System.out.println("Multiplication of all digits are : "+iRet);
    }
}